config/settings/base.py

```diff
@@ -247,3 +248,4 @@
 # Your stuff...
 # ------------------------------------------------------------------------------
 APPEND_SLASH = False
+WHITENOISE_MAX_AGE = 3600  # set Cache-Control header on static-served images
```

Comment:
Is 1hr all we want here, or can we turn it up?

Reply:
Default is 60 seconds; this might even be *too* long as it is.

The default is chosen to be short enough not to cause problems with stale versions but long enough that, if you’re running WhiteNoise behind a CDN, the CDN will still take the majority of the strain during times of heavy load.

http://whitenoise.evans.io/en/latest/django.html#WHITENOISE_MAX_AGE

Reply:
👍🏻